ui/client/components/modals/SaveProcessDialog.tsx

```diff
@@ -2,7 +2,7 @@ import React, {ChangeEvent, useCallback, useState} from "react"
 import {useTranslation} from "react-i18next"
 import {useDispatch, useSelector} from "react-redux"
 import {saveProcess} from "../../actions/nk"
-import {getProcessId, getProcessToDisplay, isProcessRenamed} from "../../reducers/selectors/graph"
+import {getProcessId, getProcessNewId, isProcessRenamed} from "../../reducers/selectors/graph"
```

Comment:
yes, "process.id" is due to "historical" reasons, let's use processName in selectors etc.

Reply:
every iteration in react needs key's for elements. in this function key is the last argument and it's most time omitted in usages. I just added some fallback to almost always set and probably unique values to avoid react warnings.